webroot/js/participantSummary.js

```diff
@@ -123,7 +164,23 @@ $(document).ready(() => {
 
         url = '/api/partner_config/get';
         doApiCallWithCallback('GET', url, (r) => { loadOptoutWebhooksCallback(r, site.name) }, (err) => { participantSummaryErrorHandler(err, '#webhooksErrorOutput') });
-    
+
+        url = `/api/sharing/keysets/related?site_id=${site.id}`;
+        doApiCallWithCallback('GET', url, (r) => { loadRelatedKeysetsCallback(r, site.id, site.clientTypes) }, (err) => { participantSummaryErrorHandler(err, '#relatedKeysetsErrorOutput') });
         $('.section').show();
     });
+
+    $('#doRotateKeysets').on('click', () => {
+        var keysets = $('#relatedKeysetsStandardOutput').text();
+        const ja = JSON.parse(keysets);
+        var rotateKeysetsMessage = '';
+        ja.forEach((keyset) => {
+            var url = `/api/key/rotate_keyset_key?min_age_seconds=3600&keyset_id=${keyset.keyset_id}&force=true`;
```

Comment:
I doublechecked the code, the min age seconds doesn't matter if we set force to true, maybe we can just set min_age_seconds=0 here

Reply:
Yes, I tried setting `min_age_seconds=0` but it will return with 404 as it needs a positive integer 😢 I can change it to 1 if that makes more sense 🤷‍♀️

Reply:
should we make `min_age_seconds=0` not return a 404 then and get rid of force?

Reply:
Lemme separate that to a new ticket as it's behaviour change 😢 and need some extra frontend work

Reply:
https://atlassian.thetradedesk.com/jira/browse/UID2-3898